### ISSUE TITLE
Add a network statistic panel

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#41]
This is currently a rough idea to be refined later but we wanted to add it to the list in case it changes architecture thinking.

We imagine two kinds of statistics may be useful to auto-calculate and display long-term:

1: attribute statistics (e.g., the sun or average value of a property)

2: network statistics such as average number of nearest neighbors, average degree, etc

We will provide examples later but wanted to get it on your radar.

Goal is for Spring or beyond.

**Comment by benloh** [comment]
This should be relatively straightforward with the new architecture.  The only caveat might be for huge networks where we might want to do some performance tuning.

**Comment by jdanish** [comment]
Makes sense! Most of the competitors actually only calculate when you push a button because it takes a while, so we can also think about where things benefit from being real-time versus delayed.
